admin/settingstree.php

```diff
@@ -0,0 +1,45 @@
+<?php
+// must be run within Dokuwiki
+if(!defined('DOKU_INC')) die();
+
+/**
+ * All DokuWiki plugins to extend the admin function
+ * need to inherit from this class
+ */
+class admin_plugin_dw2pdf_settingstree extends DokuWiki_Admin_Plugin {
+	private $helper = null;
```

Comment:
What about creating one Admin interface by the Settingtree Plugin itself, and let the settingtree plugin show a selector for each plugin supported? (see for example the ConfManager Plugin https://www.dokuwiki.org/plugin:confmanager) So not including an Admin interface in each plugin.